src/common/verbose.cpp

```diff
@@ -514,6 +514,7 @@ std::string md2fmt_str(
             break;
         case format_kind::wino:
         case format_kind::rnn_packed:
+        case format_kind::cublaslt_blocked: ss << ":col32_2r_4r4:"; break;
```

Comment:
I think this will be printed for wino and rnn_packed which wasn't the intent, I guess

Reply:
Also I think you might want to add a cublasltfmt2str function that would be called here, instead of hardcoding the string in this function.

Reply:
New function added and reporting blocked format, should make future extension easier. Addionally, fixed the case where it would report for wino and rnn_packed.